docgenerator/SDKDocGeneratorLib/Writers/BaseWriter.cs

```diff
@@ -516,15 +516,13 @@ protected void AddNamespace(TextWriter writer, string ns, string moduleName)
         protected void AddVersionInformation(TextWriter writer, AbstractWrapper wrapper)
         {
             AddSectionHeader(writer, "Version Information");
-
-            var docs35 = NDocUtilities.FindDocumentation(Artifacts.NDocForPlatform("net35"), wrapper);
-            var docs45 = NDocUtilities.FindDocumentation(Artifacts.NDocForPlatform("net45"), wrapper);
+            
+            var docs472 = NDocUtilities.FindDocumentation(Artifacts.NDocForPlatform("net472"), wrapper);
             var docsCore20 = NDocUtilities.FindDocumentation(Artifacts.NDocForPlatform("netstandard2.0"), wrapper);
             var docsNetCoreApp31 = NDocUtilities.FindDocumentation(Artifacts.NDocForPlatform("netcoreapp3.1"), wrapper);
 
             // If there is no documentation then assume it is available for all platforms.
-            var boolNoDocs = docs35 == null && docs45 == null 
-                && docsCore20 == null && docsNetCoreApp31 == null;
+            var boolNoDocs = docs472 == null && docsCore20 == null && docsNetCoreApp31 == null;
```

Comment:
Shouldn't .NET 8 target be considered for part of this?

Reply:
Yes it should. Looks like this is a bug in the existing API docs also. Will fix.

Reply:
This is already correct in main: https://github.com/aws/aws-sdk-net/blob/6e129bab6379151cf5ffaecae99a4b354fd07f51/docgenerator/SDKDocGeneratorLib/Writers/BaseWriter.cs#L527

The reason it is wrong here is because we haven't synced up changes from `main` yet into the `v4-development` branch. I will defer this change in favor of getting it from the sync.